packages/design-system/src/components/EmptyState/illustrations/SpotDefault.tsx

```diff
@@ -0,0 +1,249 @@
+			/>
+			<path
+				d="m222.76 43.902-34.87 4.247-.882-3.598 34.869-4.246.883 3.597ZM219.352 52.123l-29.606 3.605-.881-3.594 29.607-3.609.88 3.598ZM226.477 59.064l-34.872 4.249-.88-3.597 34.869-4.25.883 3.598Z"
+				fill="var(--coral-color-illustration-color-03, hsla(0,100%,74%,1))"
+			/>
+			<path
+				d="m99.82 124.238-3.949.479-4.74-19.333 3.95-.482 4.739 19.336Z"
+				fill="var(--coral-color-illustration-color-01, hsla(252,43%,25%,1))"
+			/>
+			<path
+				d="m105.725 123.519-3.951.482-3.74-15.266 3.948-.479 3.743 15.263Z"
+				fill="var(--coral-color-illustration-color-04, hsla(201,41%,64%,1))"
+			/>
+			<path
+				d="m111.695 122.79-3.951.479-7.191-29.34 3.951-.482 7.191 29.343Z"
+				fill="var(--coral-color-illustration-color-01, hsla(252,43%,25%,1))"
+			/>
+			<path
+				d="m117.6 122.071-3.95.482-6.132-25.017 3.95-.481 6.132 25.016Z"
+				fill="var(--coral-color-illustration-color-04, hsla(201,41%,64%,1))"
+			/>
+			<path
+				d="m123.575 121.341-3.951.482-3.595-14.671 3.949-.482 3.597 14.671Z"
+				fill="var(--coral-color-illustration-color-01, hsla(252,43%,25%,1))"
+			/>
+			<path
+				d="m129.476 120.622-3.949.482-5.654-23.066 3.951-.481 5.652 23.065Z"
+				fill="var(--coral-color-illustration-color-04, hsla(201,41%,64%,1))"
+			/>
+			<path
+				d="m135.448 119.898-3.949.482-7.399-30.19 3.948-.481 7.4 30.189Z"
+				fill="var(--coral-color-illustration-color-01, hsla(252,43%,25%,1))"
+			/>
+			<path
+				d="m141.352 119.18-3.948.479-5.384-21.963 3.95-.481 5.382 21.965ZM159.483 92.59a5.818 5.818 0 0 1-5.08 7.333 7.766 7.766 0 0 1-5.188-1.332 7.748 7.748 0 0 1-3.087-4.373 5.818 5.818 0 0 1 5.079-7.333 7.77 7.77 0 0 1 5.189 1.333 7.752 7.752 0 0 1 3.087 4.373Z"
+				fill="var(--coral-color-illustration-color-04, hsla(201,41%,64%,1))"
+			/>
+			<path
+				d="m183.147 102.058-33.67 4.104-.735-2.994 33.671-4.104.734 2.994ZM177.836 108.716l-26.926 3.282-.734-2.992 26.928-3.281.732 2.991ZM186.012 113.736l-33.67 4.103-.735-2.994 33.671-4.101.734 2.992ZM211.395 86.269a5.818 5.818 0 0 1-5.079 7.333 7.769 7.769 0 0 1-5.189-1.333 7.752 7.752 0 0 1-3.087-4.373 5.82 5.82 0 0 1 5.08-7.333 7.767 7.767 0 0 1 5.188 1.333 7.753 7.753 0 0 1 3.087 4.373Z"
+				fill="var(--coral-color-illustration-color-03, hsla(0,100%,74%,1))"
+			/>
+			<path
+				d="m235.057 95.736-33.67 4.101-.735-2.992 33.673-4.103.732 2.994Z"
+				fill="var(--coral-color-illustration-color-08, hsla(211,23%,74%,1))"
+			/>
+			<path
+				d="m229.744 102.394-26.928 3.281-.734-2.994 26.928-3.279.734 2.992ZM237.92 107.412l-33.67 4.101-.734-2.992 33.672-4.103.732 2.994Z"
+				fill="var(--coral-color-illustration-color-03, hsla(0,100%,74%,1))"
+			/>
+			<path
+				fillRule="evenodd"
+				clipRule="evenodd"
+				d="M204.846 47.943a10.605 10.605 0 0 0-4.369-6.272 16.134 16.134 0 0 0-5.919-2.528c-.862-.352-1.111-.054-.748.896 1.218.92 2.478 1.833 3.78 2.739-1.001.583-1.405 1.775-1.214 3.575a7.915 7.915 0 0 0 5.852 3.749l2.618-2.159Z"
+				fill="#B27F64"
```

Comment:
Should use `illustration/skintone02`, isn't it?

Reply:
Yes 🤔 I must have forgot those tokens in my script